### PR TITLE
Removing obsolete Group block wrappers

### LINKED ISSUE #587: Remove obsolete Group block wrappers
**Is your feature request related to a problem? Please describe.**

There is an obsolete Group block wrapper applied on `post-title` block in `single.html` and `single-with-sidebar.html` template.

**Describe the solution you'd like**

We can remove this Group block as it does not provide any styling or layout benefits.

### PR DESCRIPTION
**Pull request by webmandesign**
**Description**

Fixes #587

Tested with WP 6.4-beta2 with Gutenberg 16.7.1.

### CODE REVIEW

File: templates/single.html [hunk 1]
```diff
@@ -55,4 +51,4 @@
 </main>
```
Comment (luminuu) [inline templates/single.html:53]: Before the closing `main`, there's a closing group tag/comment missing:

```
</div>
<!-- /wp:group -->
```

**Review by luminuu — CHANGES_REQUESTED** [review]

File: templates/single.html (hunk 1 above)
Comment (webmandesign) [reply templates/single.html:53]: Interesting. This must have been missing for some time. I've added the closing Group tag here.

Comment (webmandesign) [reply templates/single.html:53]: Great catch!

**Review by luminuu — APPROVED** [review]
LGTM

**Pull request merged**